template.distillery/Makefile.db

```diff
@@ -30,8 +30,20 @@ ELIOMDEP          := $(ENV_PSQL) $(ELIOMDEP)
 # should be the same as ENV_PSQL)
 OCSIGENSERVER     := $(OCSIGENSERVER)
 OCSIGENSERVER.OPT := $(OCSIGENSERVER.OPT)
+
 ## ---------------------------------------------------------------------
 
+##----------------------------------------------------------------------
+
+pg_dump = pg_dump
+
+pg_ctl  = $(shell \
+           sh -c "which pg_ctl" || \
+           sh -c "which pg_ctlcluster" || \
+           sh -c "if [ -f pg_ctl ]; then (echo ./pg_ctl) else (echo '') fi")
```

Comment:
If we end up in the empty string case, the result will be very confusing error messages, no? Maybe make up an informational non-existent executable, e.g., `cannot_find_pg_ctl`.